.babelrc

```diff
@@ -13,7 +13,7 @@
   ],
   "presets": [
     "@babel/preset-react",
-    ["@babel/preset-env", { "modules": false }],
+    ["@babel/preset-env", { "modules": "auto" }],
```

Comment:
see also, comments on `ts.config.base` but:
`babel` docs say when using a bundler like webpack this should always be auto, unless we are shipping native ES modules, which I do not believe we are.